.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
You should run `git rm --cached .vscode/settings.json` to remove this from git's index. It will not remove the file from your computer in case it has been customized.